src/rest/endpoint.rs

```diff
@@ -74,6 +80,25 @@ impl Endpoint {
             }
         }
 
+        endpoints.sort_by(|a, b| {
```

Comment:
What if we have multiple files? This sorting is only going to happen within one file. Ideally we would like to do it across all the files.